content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -136,9 +136,14 @@ and set this flag to `false`. For example:
 
 ## {{% heading "whatsnext" %}}
 
-[Cron expression format](https://en.wikipedia.org/wiki/Cron)
-documents the format of CronJob `schedule` fields.
-
-For instructions on creating and working with cron jobs, and for an example of CronJob
-manifest, see [Running automated tasks with cron jobs](/docs/tasks/job/automated-tasks-with-cron-jobs).
-
+* Learn about [Pods](/docs/concepts/workloads/pods) and
+  [Jobs](/docs/concepts/workloads/controllers/job/), two concepts
+  that CronJobs rely upon.
+* Read about the [format](https://pkg.go.dev/github.com/robfig/cron/v3#hdr-CRON_Expression_Format)
```

Comment:
```suggestion
* Read about the [format](https://pkg.go.dev/github.com/robfig/cron/v3#hdr-CRON_Expression_Format/)
```
nit: trailing slash

Reply:
I'm not sure if either https://pkg.go.dev/github.com/robfig/cron/v3#hdr-CRON_Expression_Format/ or https://pkg.go.dev/github.com/robfig/cron/v3#hdr-CRON_Expression_Format is canonical. Both work and neither redirects to the other.

Reply:
Oh right. I think trailing slashes don't work for subsection references 🤔